README.md

```diff
@@ -46,7 +47,7 @@ This will show the logs from the OpenShift API, there should be some information
 
 To allow the `admin` user from Keycloak to run `oc get sa` run the following:
 
-    oc --token=$(kcinit token) get sa
+    ./oc --token=$(kcinit token) get sa
```

Comment:
Using ./oc gets old fast, why not simply advice people to download the oc command in ~/bin or add the directory in which it is download to the PATH ?

Reply:
That could also be a good way. @stianst WDYT?